### DISCUSSION TITLE
The 2.5 lecture slides switch to Java?

### DISCUSSION THREAD

**Discussion by jdonwells** [TEALSK12/introduction-to-computer-science#335] [Ideas]
The 2.5 lecture slides seem to use Snap! code up until slide 14 and then it goes to Java or Javascript or something.

I think we should continue to use Snap! code since that is what they know.
![Screenshot 2021-09-26 2 07 33 PM](https://user-images.githubusercontent.com/5577101/134819277-57d609a3-d43a-4dcf-9fc8-713b25e3afce.png)
![Screenshot 2021-09-27 1 19 25 PM](https://user-images.githubusercontent.com/5577101/134956084-44b16db9-06c9-44eb-a3ea-b47841a6f2f2.png)

![Screenshot 2021-09-26 2 04 10 PM](https://user-images.githubusercontent.com/5577101/134819287-e7987058-7106-4267-8587-9224dc8339b1.png)

PS now that they have conditionals it would be instructive to show them Yellow Brick Road with **repeat until** loops.
[https://snap.berkeley.edu/snap/snap.html#present:Username=jdonwells&ProjectName=Lab%202.2%20Yellow%20Brick%20Road](https://snap.berkeley.edu/snap/snap.html#present:Username=jdonwells&ProjectName=Lab%202.2%20Yellow%20Brick%20Road)
It is the only way to solve it will minimal code and no extra bricks.
.

**Comment by DBashirTEALS** [comment]
Thanks for mentioning this.  I'll have our curriculum team investigate further.

Dr. Bashir
